src/main/java/de/minebench/syncinv/SyncInv.java

```diff
@@ -401,16 +403,13 @@ public boolean onCommand(CommandSender sender, Command cmd, String label, String
 
     /**
      * Get a language message from the config and replace variables in it
-     * @param key The key of the message (lang.<key>)
-     * @param replacements An array of variables to be replaced with certain strings in the format [var,repl,var,repl,...]
+     * @param key          The key of the message (lang.<key>)
+     * @param replacements An array of String to replace placeholders (uses the % character as placeholder indicators (suffix and prefix))
      * @return The message string with colorcodes and variables replaced
      */
-    public String getLang(String key, String... replacements) {
-        String msg = ChatColor.translateAlternateColorCodes('&', getConfig().getString("lang." + key, getName() + ": &cMissing language key &6" + key));
-        for (int i = 0; i + 1 < replacements.length; i += 2) {
-            msg = msg.replace("%" + replacements[i] + "%", replacements[i+1]);
-        }
-        return msg;
+    public Component getLang(String key, String... replacements) {
+        String rawMsg = getConfig().getString("lang." + key, getName() + ": <red>Missing language key</red><gold> " + key+"</gold>");
```

Comment:
The fallback still has mini message formatting, should either be in MineDown or just keep the legacy format like before (as MineDown parses that as well)